infrastructure/aat.tfvars

```diff
@@ -1,5 +1,6 @@
 management_security_enabled = "false"
-infrastructure_env = "test"
+infrastructure_env = "preprod"
 gaps2_sftp_user = "sscs-sftp-test"
-core_case_data_user_id = "43132"
+core_case_data_user_id = "506"
```

Comment:
where did you get that id from?

Reply:
From TCA api configuration for demo which from the beginning was pointing to preprod so same endpoint as AAT is currently using